clusterloader2/pkg/modifier/modifier.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package modifier
+
+import (
+	"k8s.io/perf-tests/clusterloader2/api"
+	"k8s.io/perf-tests/clusterloader2/pkg/config"
+	"k8s.io/perf-tests/clusterloader2/pkg/flags"
+)
+
+// Modifier mutates provided test
+type Modifier interface {
+	ChangeTest(*api.Config)
+}
+
+// InitFlags allows setting configuration with flags
+func InitFlags(m *config.ModifierConfig) {
+	flags.StringArrayVar(&m.SkipSteps, "skip-steps", []string{}, "Name of steps to skip in test")
+}
+
+// NewModifier creates new Modifier according to provided configuration
+func NewModifier(m *config.ModifierConfig) Modifier {
+	return &simpleModifier{skipSteps: m.SkipSteps}
+}
+
+type simpleModifier struct {
+	skipSteps []string
```

Comment:
NIT: I'd make this field of type map[string]bool to have a O(1) lookup, but given the # steps is usually < 20-30, I don't think it would change a lot :)

Reply:
Thanks for the tip!

Ideally, flags would support maps as a first class citizens, then it would be easier to use maps when they are better choice.